go/extensions/types.go

```diff
@@ -1,12 +1,75 @@
 package extensions
 
 import (
-    sdk "github.com/cosmos/amino-js/go/lib/cosmos/cosmos-sdk/types"
+	bridge "github.com/cosmos/amino-js/go/lib/bridge/types"
+	sdk "github.com/cosmos/amino-js/go/lib/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/amino-js/go/lib/tendermint/tendermint/libs/common"
+	tm "github.com/cosmos/amino-js/go/lib/tendermint/tendermint/types"
 )
 
 var _ sdk.Msg = (*TxCreateMarket)(nil)
 
 type TxCreateMarket struct {
-   Account sdk.AccAddress
-   Market string
+	Account sdk.AccAddress
+	Market  string
+}
+
+var _ sdk.Tx = (*BridgeTx)(nil)
+
+type BridgeTx struct {
+	Hash     string           `json:"hash"`
+	Height   int64            `json:"height"`
+	Index    uint32           `json:"index"`
+	TxResult BridgeResponseTx `json:"tx_result"`
+	Tx       tm.Tx            `json:"tx"`
+}
+
+type BridgeResponseTx struct {
+	Code                 uint32          `protobuf:"varint,1,opt,name=code,proto3" json:"code,omitempty"`
+	Data                 []byte          `protobuf:"bytes,2,opt,name=data,proto3" json:"data,omitempty"`
+	Log                  string          `protobuf:"bytes,3,opt,name=log,proto3" json:"log,omitempty"`
+	Info                 string          `protobuf:"bytes,4,opt,name=info,proto3" json:"info,omitempty"`
+	GasWanted            int64           `protobuf:"varint,5,opt,name=gas_wanted,json=gasWanted,proto3" json:"gas_wanted,omitempty"`
+	GasUsed              int64           `protobuf:"varint,6,opt,name=gas_used,json=gasUsed,proto3" json:"gas_used,omitempty"`
+	Events               []common.KVPair `protobuf:"bytes,7,rep,name=events" json:"events,omitempty"`
+	Codespace            string          `protobuf:"bytes,8,opt,name=codespace,proto3" json:"codespace,omitempty"`
+	XXX_NoUnkeyedLiteral struct{}        `json:"-"`
```

Comment:
I'm not sure if the `XXX_`-prefixed keys will be used anywhere since they don't end up in the JSON output.

Reply:
https://github.com/denalimarsh/amino-js/pull/2/commits/cecdc355c128f39ca475006791031e59fdb102a3